server/http_handler.go

```diff
@@ -744,6 +746,13 @@ func NewFrameItemFromRegionKey(key []byte) (frame *FrameItem, err error) {
 	frame = &FrameItem{}
 	frame.TableID, frame.IndexID, frame.IsRecord, err = tablecodec.DecodeKeyHead(key)
 	if err == nil {
+		if frame.IsRecord {
+			_, frame.RecordID, err = tablecodec.DecodeRecordKey(key)
+		} else {
+			_, _, frame.IndexValues, err = tablecodec.DecodeIndexKey(key)
+		}
+		// Ignore decode errors.
+		err = nil
```

Comment:
The err would always be nil in this function, so could we remove the err in response? Could we print some warning if any error happens?

Reply:
ok. I'm going to add an warning here.